src/kvstore/Listener.cpp

```diff
@@ -209,7 +209,8 @@ void Listener::doApply() {
           break;
         }
         default: {
-          LOG(WARNING) << idStr_ << "Unknown operation: " << static_cast<int32_t>(log[0]);
+          VLOG(2) << idStr_
```

Comment:
since it is some kind of exception, what's the reason of changing from log warning to vlog?

Reply:
Two reasons here:
The max log level is limited to VLOG(1)
If the exception happens, log here will be printed every time for each log entry, which is unexpectd.